java/ql/src/experimental/Security/CWE/CWE-094/TemplateInjection.qll

```diff
@@ -0,0 +1,247 @@
+/** Definitions related to the Server Side Template Injection (SSTI) query. */
+
+import java
+import semmle.code.java.dataflow.TaintTracking
+import semmle.code.java.dataflow.FlowSources
+import experimental.semmle.code.java.frameworks.FreeMarker
+import experimental.semmle.code.java.frameworks.Velocity
+import experimental.semmle.code.java.frameworks.MVEL
+import experimental.semmle.code.java.frameworks.JinJava
+import experimental.semmle.code.java.frameworks.Pebble
+import experimental.semmle.code.java.frameworks.Thymeleaf
+
+/** A module to reason about Server Side Template Injection (SSTI) vulnerabilities */
+module TemplateInjection {
```

Comment:
No need for a module surrounding the whole file

Reply:
There are some methods here which make sense only for this particular query. These have been marked private. If I remove the module surrounding them, won't I unnecessarily pollute the global namespace?

Reply:
Removed! PTAL.